polaris-core/src/main/java/org/apache/polaris/core/persistence/MetaStoreManagerFactory.java

```diff
@@ -19,33 +19,26 @@
 package org.apache.polaris.core.persistence;
 
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
-import io.dropwizard.jackson.Discoverable;
 import java.util.List;
 import java.util.Map;
 import java.util.function.Supplier;
 import org.apache.polaris.core.auth.PolarisSecretsManager.PrincipalSecretsResult;
 import org.apache.polaris.core.context.RealmContext;
-import org.apache.polaris.core.monitor.PolarisMetricRegistry;
-import org.apache.polaris.core.storage.PolarisStorageIntegrationProvider;
 import org.apache.polaris.core.storage.cache.StorageCredentialCache;
 
 /**
  * Configuration interface for configuring the {@link PolarisMetaStoreManager} via Dropwizard
  * configuration
  */
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
```

Comment:
While this and #469 aim to migrate to "proper CDI", the use of DI via Jackson should be removed as well.

Reply:
Yeah, I think I missed a handful of these. I'll go back and revisit existing types.